contrib/freshmaker-cli

```diff
@@ -398,7 +458,12 @@ if not api_url:
     print("There is no Freshmaker instance for deployment %s and env %s" %( deployment, env))
     sys.exit(1)
 
-if not args.events:
+if args.errata:
```

Comment:
Could you please print an error and exit if both `args.events` and `args.errata` are set?